operator/pkg/helmreconciler/prune.go

```diff
@@ -147,6 +148,14 @@ func (h *HelmReconciler) DeleteObjectsList(objectsList []*unstructured.Unstructu
 					h.opts.Log.LogAndPrintf("object: %s is not being deleted because it no longer exists",
 						obj.Hash())
 				}
+			} else {
+				objGvk := o.GroupVersionKind()
+				gvkStr := fmt.Sprintf("%s/%s/%s", objGvk.Group, objGvk.Version, objGvk.Kind)
+				revision := h.iop.Spec.Revision
+				iopName := fmt.Sprintf("%s/%s", h.iop.GetNamespace(), h.iop.GetName())
+				metrics.CountResourceDeletions(iopName, revision, gvkStr)
```

Comment:
on reflection i think the cardinality may get too high with gvkstr. maybe drop that label? it's not super useful for deletes.

Reply:
Done